main.js

```diff
@@ -202,4 +199,88 @@ function hexToRgba(hex) {
     parseInt(hex.slice(3, 5), 16),
     parseInt(hex.slice(5, 7), 16),
     255];
-}
\ No newline at end of file
+}
+
+/**
+ * Convert an sRGB color to CIELAB
+ * http://www.easyrgb.com/en/math.php
```

Comment:
It's not clear to me what CIELAB is from this reference -- is there a better reference to link here?

Reply:
Would also appreciate a reference to what XYZ and LAB are, thanks.

Reply:
Added a link to an easy explanation and wikipedia articles for all the new terminologies.